ui/components/app/wallet-overview/wallet-overview.js

```diff
@@ -24,6 +33,16 @@ const WalletOverview = ({
           </Box>
         ) : null}
         {balance}
+        {label ? (
+          <Tag
+            label={label}
+            labelProps={{
+              variant: TextVariant.bodyXs,
+              color: Color.textAlternative,
+            }}
+            startIconName={keyring === KeyringType.snap ? IconName.Snaps : null}
```

Comment:
Based on the previous suggestion, this should be update like this
```suggestion
            startIconName={keyring.type === KeyringType.snap ? IconName.Snaps : null}
```